AVL.cabal

```diff
@@ -58,7 +58,7 @@ library
                MultiParamTypeClasses
                MultiWayIf
                NamedFieldPuns
-               OverloadedStrings
+--             OverloadedStrings
```

Comment:
1) Why?
2) Why comment if you might just delete?

Reply:
1. Because I see `OverloadedStrings` as adding nothing that nuisance.
2. Sry, bad habit.